common/annotations/en.xml

```diff
@@ -250,8 +250,8 @@ annotations.
 		<annotation cp="↷" type="tts">clockwise top semicircle arrow</annotation>
 		<annotation cp="↸">north west arrow long bar</annotation>
 		<annotation cp="↸" type="tts">north west arrow long bar</annotation>
-		<annotation cp="↹">leftwards arrow bar over rightwards arrow bar</annotation>
-		<annotation cp="↹" type="tts">leftwards arrow bar over rightwards arrow bar</annotation>
+		<annotation cp="↹">leftwards | arrow | bar | rightwards</annotation>
+		<annotation cp="↹" type="tts">leftwards over rightwards arrow bars</annotation>
```

Comment:
The addition of vertical line characters here differs from the usual pattern in this file. Does it matter?

Reply:
That is a problem in the new symbols. I made a ticket for Andrew to take to fix them, but went ahead with this one.

https://unicode-org.atlassian.net/browse/CLDR-14859